src/endpoint/blob/blob_utils.js

```diff
@@ -53,6 +55,58 @@ function parse_copy_source(req) {
 }
 
 
+
+
+//////// BLOB API UTILS ////////
+
+async function list_objects(params, account_name, container, sasToken) {
```

Comment:
missing function doc - why does this exist, where is it taken from, etc.

Reply:
I wrote why it exist in the function call and that it's taken from the API. should I add/move it to this file ?